Tools/CISupport/ews-build/steps.py

```diff
@@ -3824,10 +3834,10 @@ def send_email_for_new_test_failures(self, test_names, exceed_failure_limit=Fals
                 change_author, errors = GitHub.email_for_owners(self.getProperty('owners', []))
                 for error in errors:
                     print(error)
-                    self._addToLog('stdio', error)
+                    yield self._addToLog('stdio', error)
 
             if not change_author:
-                self._addToLog('stderr', 'Unable to determine email address for {} from metadata/contributors.json. Skipping sending email.'.format(self.getProperty('owners', [])))
+                yield self._addToLog('stderr', 'Unable to determine email address for {} from metadata/contributors.json. Skipping sending email.'.format(self.getProperty('owners', [])))
                 return
```

Comment:
Do we need defer.returnValue(None) here?

Reply:
See above comment, no, that's the default.